tests/Microsoft.Identity.Test.Integration.netcore/HeadlessTests/ClientCredentialsMtlsPopTests.cs

```diff
@@ -22,15 +22,15 @@ namespace Microsoft.Identity.Test.Integration.HeadlessTests
     [TestClass]
     public class ClientCredentialsMtlsPopTests
     {
-        private static readonly string[] s_scopes = { "User.Read" };
-        private const string LabAccessConfidentialClientId = "f62c5ae3-bf3a-4af5-afa8-a68b800396e9";
+        private const string MsiAllowListedAppIdforSNI = "163ffef9-a313-45b4-ab2f-c7e2f5e0e23e";
 
         [TestInitialize]
         public void TestInitialize()
         {
             TestCommon.ResetInternalStaticCaches();
         }
 
+        [IgnoreOnClassicPipeline]
```

Comment:
Can you pls add a comment on why the test doesn't run on Classic pipeline?

Reply:
this made it easier to test in testslice for troubleshooting. removed now but would like to leave the IgnoreOnClassicPipeline.cs in there to help with other features